[2018-11-10 09:01:45,736] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.breakpoint_sensor_run_this_last_parallel 2018-11-10T13:10:00+00:00 [queued]>
[2018-11-10 09:01:45,761] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.breakpoint_sensor_run_this_last_parallel 2018-11-10T13:10:00+00:00 [queued]>
[2018-11-10 09:01:45,761] {models.py:1547} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 1
--------------------------------------------------------------------------------

[2018-11-10 09:01:45,772] {models.py:1569} INFO - Executing <Task(BreakPointSensor): breakpoint_sensor_run_this_last_parallel> on 2018-11-10T13:10:00+00:00
[2018-11-10 09:01:45,772] {base_task_runner.py:124} INFO - Running: ['bash', '-c', u'airflow run naveed_test breakpoint_sensor_run_this_last_parallel 2018-11-10T13:10:00+00:00 --job_id 149 --raw -sd DAGS_FOLDER/test.py --cfg_path /var/folders/zk/29b_jkbx6z58gzz4zv4tzd_40000gn/T/tmpZT_fnI']
[2018-11-10 09:01:49,637] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:49,637] {settings.py:174} INFO - setting.configure_orm(): Using pool settings. pool_size=5, pool_recycle=3600
[2018-11-10 09:01:49,672] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel /Users/naveed/anaconda2/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-11-10 09:01:49,672] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   """)
[2018-11-10 09:01:49,934] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:49,934] {__init__.py:51} INFO - Using executor LocalExecutor
[2018-11-10 09:01:50,095] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel /Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/bin/cli.py:1595: DeprecationWarning: The celeryd_concurrency option in [celery] has been renamed to worker_concurrency - the old setting has been used, but please update your config.
[2018-11-10 09:01:50,096] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   default=conf.get('celery', 'worker_concurrency')),
[2018-11-10 09:01:50,189] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,188] {cli.py:464} ERROR - Section ldap Option bind_user does not exist in the config!
[2018-11-10 09:01:50,189] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,188] {cli.py:464} ERROR - Section ldap Option search_scope does not exist in the config!
[2018-11-10 09:01:50,190] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,189] {cli.py:464} ERROR - Section ldap Option basedn does not exist in the config!
[2018-11-10 09:01:50,190] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,189] {cli.py:464} ERROR - Section ldap Option group_member_attr does not exist in the config!
[2018-11-10 09:01:50,191] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,189] {cli.py:464} ERROR - Section ldap Option uri does not exist in the config!
[2018-11-10 09:01:50,191] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,189] {cli.py:464} ERROR - Section ldap Option data_profiler_filter does not exist in the config!
[2018-11-10 09:01:50,192] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,189] {cli.py:464} ERROR - Section ldap Option bind_password does not exist in the config!
[2018-11-10 09:01:50,192] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section ldap Option cacert does not exist in the config!
[2018-11-10 09:01:50,193] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section ldap Option user_filter does not exist in the config!
[2018-11-10 09:01:50,193] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section ldap Option user_name_attr does not exist in the config!
[2018-11-10 09:01:50,194] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section ldap Option superuser_filter does not exist in the config!
[2018-11-10 09:01:50,194] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section atlas Option username does not exist in the config!
[2018-11-10 09:01:50,195] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,190] {cli.py:464} ERROR - Section atlas Option host does not exist in the config!
[2018-11-10 09:01:50,195] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,191] {cli.py:464} ERROR - Section atlas Option password does not exist in the config!
[2018-11-10 09:01:50,196] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,191] {cli.py:464} ERROR - Section atlas Option port does not exist in the config!
[2018-11-10 09:01:50,196] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,191] {cli.py:464} ERROR - Section atlas Option sasl_enabled does not exist in the config!
[2018-11-10 09:01:50,197] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,191] {cli.py:464} ERROR - Section kubernetes Option logs_volume_subpath does not exist in the config!
[2018-11-10 09:01:50,197] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,191] {cli.py:464} ERROR - Section kubernetes Option git_repo does not exist in the config!
[2018-11-10 09:01:50,198] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,192] {cli.py:464} ERROR - Section kubernetes Option git_user does not exist in the config!
[2018-11-10 09:01:50,198] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,192] {cli.py:464} ERROR - Section kubernetes Option worker_container_repository does not exist in the config!
[2018-11-10 09:01:50,198] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,192] {cli.py:464} ERROR - Section kubernetes Option image_pull_secrets does not exist in the config!
[2018-11-10 09:01:50,199] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,193] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_tag does not exist in the config!
[2018-11-10 09:01:50,199] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,193] {cli.py:464} ERROR - Section kubernetes Option in_cluster does not exist in the config!
[2018-11-10 09:01:50,200] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,193] {cli.py:464} ERROR - Section kubernetes Option namespace does not exist in the config!
[2018-11-10 09:01:50,200] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,193] {cli.py:464} ERROR - Section kubernetes Option dags_volume_subpath does not exist in the config!
[2018-11-10 09:01:50,201] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,193] {cli.py:464} ERROR - Section kubernetes Option git_sync_init_container_name does not exist in the config!
[2018-11-10 09:01:50,201] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option dags_volume_claim does not exist in the config!
[2018-11-10 09:01:50,202] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option git_subpath does not exist in the config!
[2018-11-10 09:01:50,202] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option worker_service_account_name does not exist in the config!
[2018-11-10 09:01:50,202] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option git_branch does not exist in the config!
[2018-11-10 09:01:50,203] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option logs_volume_claim does not exist in the config!
[2018-11-10 09:01:50,203] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option gcp_service_account_keys does not exist in the config!
[2018-11-10 09:01:50,203] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option airflow_configmap does not exist in the config!
[2018-11-10 09:01:50,204] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,194] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_repository does not exist in the config!
[2018-11-10 09:01:50,204] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,195] {cli.py:464} ERROR - Section kubernetes Option worker_container_tag does not exist in the config!
[2018-11-10 09:01:50,205] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,195] {cli.py:464} ERROR - Section kubernetes Option git_password does not exist in the config!
[2018-11-10 09:01:50,205] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,195] {cli.py:464} ERROR - Section kubernetes Option delete_worker_pods does not exist in the config!
[2018-11-10 09:01:50,205] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section lineage Option backend does not exist in the config!
[2018-11-10 09:01:50,206] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section hive Option default_hive_mapred_queue does not exist in the config!
[2018-11-10 09:01:50,206] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section dask Option cluster_address does not exist in the config!
[2018-11-10 09:01:50,207] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section dask Option tls_cert does not exist in the config!
[2018-11-10 09:01:50,207] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section dask Option tls_key does not exist in the config!
[2018-11-10 09:01:50,208] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section dask Option tls_ca does not exist in the config!
[2018-11-10 09:01:50,208] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,196] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_host does not exist in the config!
[2018-11-10 09:01:50,209] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,197] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_log_id_template does not exist in the config!
[2018-11-10 09:01:50,209] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,197] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_end_of_log_mark does not exist in the config!
[2018-11-10 09:01:50,209] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,205] {models.py:258} INFO - Filling up the DagBag from /Users/naveed/Mira/hackathon/airflow/dags/test.py
[2018-11-10 09:01:50,321] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,321] {example_kubernetes_operator.py:54} WARNING - Could not import KubernetesPodOperator: No module named kubernetes
[2018-11-10 09:01:50,322] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:50,321] {example_kubernetes_operator.py:55} WARNING - Install kubernetes dependencies with:     pip install airflow['kubernetes']
[2018-11-10 09:01:55,541] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel [2018-11-10 09:01:55,540] {cli.py:492} INFO - Running <TaskInstance: naveed_test.breakpoint_sensor_run_this_last_parallel 2018-11-10T13:10:00+00:00 [running]> on host naveeds-MacBook-Pro.local
[2018-11-10 12:50:47,243] {logging_mixin.py:95} INFO - [2018-11-10 12:50:47,241] {jobs.py:2680} WARNING - State of this instance has been externally set to failed. Taking the poison pill.

[2018-11-10 12:50:47,263] {helpers.py:240} INFO - Sending 15 to GPID 55419
[2018-11-10 12:50:47,263] {models.py:1610} ERROR - Received SIGTERM. Terminating subprocesses.
[2018-11-10 12:50:47,273] {models.py:1736} ERROR - Task received SIGTERM signal
Traceback (most recent call last):
  File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/models.py", line 1633, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/sensors/base_sensor_operator.py", line 78, in execute
    sleep(self.poke_interval)
  File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/models.py", line 1612, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
AirflowException: Task received SIGTERM signal
[2018-11-10 12:50:47,275] {models.py:1764} INFO - Marking task as FAILED.
[2018-11-10 12:50:47,296] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel Traceback (most recent call last):
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/bin/airflow", line 32, in <module>
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     args.func(args)
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     return f(*args, **kwargs)
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/bin/cli.py", line 498, in run
[2018-11-10 12:50:47,297] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     _run(args, dag, ti)
[2018-11-10 12:50:47,298] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/bin/cli.py", line 402, in _run
[2018-11-10 12:50:47,298] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     pool=args.pool,
[2018-11-10 12:50:47,298] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2018-11-10 12:50:47,298] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     return func(*args, **kwargs)
[2018-11-10 12:50:47,298] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/models.py", line 1633, in _run_raw_task
[2018-11-10 12:50:47,299] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     result = task_copy.execute(context=context)
[2018-11-10 12:50:47,299] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/sensors/base_sensor_operator.py", line 78, in execute
[2018-11-10 12:50:47,299] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     sleep(self.poke_interval)
[2018-11-10 12:50:47,299] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel   File "/Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/models.py", line 1612, in signal_handler
[2018-11-10 12:50:47,299] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel     raise AirflowException("Task received SIGTERM signal")
[2018-11-10 12:50:47,300] {base_task_runner.py:107} INFO - Job 149: Subtask breakpoint_sensor_run_this_last_parallel airflow.exceptions.AirflowException: Task received SIGTERM signal
[2018-11-10 12:50:47,375] {helpers.py:230} INFO - Process psutil.Process(pid=55419 (terminated)) (55419) terminated with exit code 1
[2018-11-10 12:50:47,376] {logging_mixin.py:95} INFO - [2018-11-10 12:50:47,375] {jobs.py:2612} INFO - Task exited with return code 0

